lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1446,3 +1489,154 @@ class EditableTextGeometry {
       ..transform = cssTransform;
   }
 }
+
+/// Controls the capitalization of the text.
+///
+/// This corresponds to Flutter's [TextCapitalization].
+///
+/// Uses `text-transform` css property.
+/// See: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
+enum TextCapitalization {
```

Comment:
May I suggest that we move this enum and the class below to their own file called `text_capitalization.dart` or something similar?

Reply:
done.

Reply:
i removed the new file. there was a license issue with fuchsia files.

Reply:
Btw these are the [steps](https://github.com/flutter/engine/tree/master/tools/licenses) that I run. I used it 3-4 times so far. Please let me know if there are any updates not represented in the documentation about license fixes.

Reply:
Actually, I solved it :) since I was not changing the license itself I didn't need to run the last step which creates failures in fuchsia. So I added the new file :)